src/components/navigation/DynamicButtons.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation, useNavigate } from 'react-router-dom';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+
+//material ui
+import Button from '@mui/material/Button';
+
+import { StyledBtn } from '../shared/StyledButtons';
+
+// State Managers
+import { useDispatch } from 'react-redux';
+// To decode the token
+import decode from 'jwt-decode';
+//login User logic for rendering
+const DynamicButtons = () => {
+  const dispatch = useDispatch();
+  //const location = useLocation();
+  const navigate = useNavigate();
+
+  //usersData set with reducer
+  //this is required to observe redux store and render the component when store has changed.
+  const user = useSelector((state) => {
+    return state.authReducer.authData;
+  });
+
+  const logout = () => {
+    dispatch({ type: 'LOGOUT' });
+    navigate('/auth');
```

Comment:
Nat, please change the route to `navigate('/')` -- otherwise it will redirect to an error page since we dont have `/auth` route

Reply:
ok.. I changed it!